packages/babel-plugin/package.json

```diff
@@ -1,18 +1,32 @@
 {
     "name": "@solenoid/babel-plugin",
+    "type": "module",
     "version": "0.0.0",
     "description": "Babel plugin for Solenoid",
     "main": "lib/index.js",
     "types": "lib/index.d.ts",
-    "repository": "https://github.com/nmn/solenoid",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/nmn/solenoid.git",
+        "directory": "packages/babel-plugin"
+    },
     "license": "MIT",
     "scripts": {
-      "build": "find ./src -name '*.ts' -exec sh -c 'bun build {} --outfile ./lib/$(basename {} .ts).js --target node --no-bundle' \\;",
-      "test": "vitest"
+        "build": "find ./src -name '*.ts' -exec sh -c 'bun build {} --outfile ./lib/$(basename {} .ts).js --target node --no-bundle' \\;",
```

Comment:
You've messed up the formatting...

Reply:
Oomf, I'll fix

Reply:
I didn't do this either, I believe bun reformatted the package.json. It's using 4 spaces for the tab width, which is consistent with the rest of the file.

Do we want to make it 2 spaces?